docs/source/using/managing-minishift.adoc

```diff
@@ -275,3 +275,29 @@ Use the `openshift-version` option to request a specific version of OpenShift. Y
 
 The Minishift VM is exposed to the host system with a host-only IP address that
 can be obtained with the `minishift ip` command.
+
+[[connecting-with-ssh]]
+== Connecting to the Minishift VM with SSH
+
+You can use the xref:../command-ref/minishift_ssh{outfilesuffix}[`minishift ssh`] command to interact with the Minishift VM.
+
+When you run this command, it checks whether you have an external SSH client installed on your machine. An SSH client is not a
+prerequisite, but it impacts the behavior of the command.
+
+If you have an external SSH client::
+You can run `minishift ssh` without a sub-command to open an interactive shell using your external SSH client.
+Then, you can run commands on the Minishift VM in the same way that you run commands interactively on any remote machine using SSH.
+
+You can also run the command with a sub-command. The native Minishift client then sends the command to the Minishift VM
```

Comment:
native is the Go implementation, as we observed different behaviour between the use of the external client, and the way docker-machine can provide this with the Go's native implementation. However, it is believed there is an issue with the way how minikube/minishift work => #1005